CliFx/Domain/CommandArgumentSchema.cs

```diff
@@ -17,6 +17,8 @@ internal abstract partial class CommandArgumentSchema
 
         public bool IsScalar => TryGetEnumerableArgumentUnderlyingType() == null;
 
+        protected Type? Converter { get; set; }
```

Comment:
Can this be set from constructor like the rest of the properties?

Reply:
done